submission/persistence-mongo/src/main/kotlin/ac/uk/ebi/biostd/persistence/doc/db/data/SubmissionDocDataRepository.kt

```diff
@@ -154,21 +140,42 @@ class SubmissionDocDataRepository(
             offsetLimit?.let { add(skip(it.first)); add(limit(it.second)) }
         }
 
+        @Suppress("ComplexMethod")
         private fun createQuery(filter: SubmissionFilter): List<MatchOperation> {
+            fun sectionTitleContains(keywords: String) =
+                where("$SUB_SECTION.$SUB_ATTRIBUTES").elemMatch(
+                    Criteria().andOperator(
+                        where("name").`is`("Title"),
```

Comment:
Since we have [constant definitions for all these fields](https://github.com/EBIBioStudies/biostudies-backend-services/blob/master/commons/commons-bio/src/main/kotlin/ebi/ac/uk/model/constants/Fields.kt) I think it'd be better to use them instead of hardcoding them here